mission_control/README.md

```diff
@@ -0,0 +1,22 @@
+# Mission Control
+The mission control package provides an implementation of missions using behavior trees
+It is compliant with the the [BehaviorTree.CPP](https://github.com/BehaviorTree/BehaviorTree.CPP) library.  
+
+## Features
+
+- The mission control implements the interface with the Jaus Ros Bridge ROS Package.
+
+The behaviors/actions implemented were:
+- Abort
+- Attitude Servo
+- Fix Rudder
+- Go To Waypoint
+- Payload Command
+- Set Altitude Heading
+- Set Depth Heading
+
+- The missions can be designed using Groot
```

Comment:
@patapufate I think one bullet should talk about missions being described as behavior trees in XML.

Reply:
Done